comps/cores/mega/orchestrator.py

```diff
@@ -301,6 +305,7 @@ def extract_chunk_str(self, chunk_str):
             chunk_str = chunk_str[: -len(suffix)]
         return chunk_str
 
+    @opea_telemetry
     def token_generator(self, sentence: str, token_start: float, is_first: bool, is_last: bool) -> str:
```

Comment:
This is case 2). As shown from your Jaeger UI image, I'm afraid this is never traced?